lib/lightspeed/client.rb

```diff
@@ -21,19 +23,23 @@ def accounts
     end
 
     def get(**args)
-      perform_request(args.merge(method: :get))
+      args.merge!(method: :get)
+      perform_request(**args)
     end
 
     def post(**args)
-      perform_request(args.merge(method: :post))
+      args.merge!(method: :post)
+      perform_request(**args)
```

Comment:
this could be `perform_request(**args, method: :post)` etc

Reply:
Much nicer, Thanks. (and changed)